solvestack/glossary/migrations/0002_stacknode.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 3.1.2 on 2020-10-24 19:29
+
+from django.db import migrations, models
+import django_extensions.db.fields
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('glossary', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='StackNode',
```

Comment:
Oh, when you change a model name you either have to create another migration that renames the model, OR if it hasn't made it to the master or main branch yet you can just change the name in the migration itself.

Reply:
Good catch!  Modified the migration file and pushed.  Dumb question... but would removing the migration folder with .gitignore make the whole process happen without inheriting the history of changes?  Or are we kinda stuck with these because we started w/ 'em and need to work them out this way?